callback_function.go

```diff
@@ -6,10 +6,10 @@ package lexfloatclient
 #include <stdio.h>
 
 // The gateway functions
-void licenseCallbackCgoGateway(int status)
+void floatingLicenseCallbackCgoGateway(int status)
```

Comment:
i think we should also rename this file callback_function to floating_callback_function

Reply:
Although it works fine at this stage as well. Tested it locally and now the error does not occur. But now that we have used floating with the callback we should rename the file as well.

Reply:
Yes, please update the name